README.md

```diff
@@ -1,28 +1,70 @@
 Instagram Challenge
 ===================
 
-Instructions
+About
 -------
-* Challenge time: Friday, the entire day + the weekend if you need it
-* Feel free to use google, your notes, books, etc but work on your own
-* You must submit a pull request to this repo with your challenge solution (or partial solution) by 9am Monday morning
+2 days to recreate instagram from scratch using rails!
 
-Task
+Screenshot
 -----
 
-Build Instagram: Simple huh!
+![screenshot](./instaham/public/Screenshot.png)
 
-As usual please start by
 
-* Filling out your learning plan self review for the week: https://github.com/makersacademy/learning_plan (if you haven't already)
-* Forking this repo
+User Stories Extracted From Brief
+-----
+
+As a ham loving youth
+So that i can share my joy
+I would like to post pictures of ham
+
+As a ham loving youth
+So that i can express my excitement at other peoples ham picutres
+I would like to comment on pictures
+
+As a expressive youth
+So that i can commincate my pleasure
+I would like to be able to click like on a picture
+
+As a busines owner
+to maximise ROI
+I would like to limit posting privaliges to signed up users
+
+MVps
+-----
+
+1: Able to post a message to site
+
+2: Able to limit posting messages to signed up users
+
+3: can comment on photos
+
+4: messages are now photos stored in the cloud
+
+5: awesome design
+
+6: can like photos
+
+
+Next Steps
+-----
+
+Plan to fix issue with bootstrap gem causing tests to fail, and expand, with user names and timestamps on comments, and ability to delete accounts/photos.
+
+Will also extend design to sign in page and push to heroku.
+
+How to contribute
+-----
+
+Thoughts, ideas and oppinions always welcome, simply leave a comment! :D
+
+How to run
+-----
+
+Git clone to your machine.
+
+In the root, run 'bundle', then 'rake db:setup' then 'rails s'
```

Comment:
Three backticks (```) will give you block-level monospaced stuff (i.e. code blocks):

In the root, install dependencies:

```bash
bundle install
```
set up the database:
```bash
rake db:setup
```
and run the server:
```bash
rails server
```

(try to use full commands instead of shortened ones (e.g. `rails s`) in case your users aren't familiar with them!)